plugins/registry.js

```diff
@@ -767,8 +767,10 @@ export default class PluginRegistry {
     // - switcherText - A string or React element to display in the product switcher
     // - switcherLinkURL - A string specifying the URL the switcher item should point to.
     // - mainComponent - The component to be displayed below the global header when your route is active.
-    // - headerComponent - A component to fill the generic area in the center of
-    //                     the global header when your route is active.
+    // - headerCentreComponent - A component to fill the generic area in the center of
```

Comment:
Not something to change here, but we probably want to rename this to "center" to keep American spelling throughout the code base. Thankfully it doesn't matter here since this is just the name of an argument that doesn't affect users